lib/init.ts

```diff
@@ -84,3 +92,7 @@ You can visit https://algolia.com/events/debugger instead.`);
     this.setAnonymousUserToken();
   }
 }
+
+function patchOption<TOption>(option: TOption, fallback: TOption) {
+  return isUndefined(option) ? fallback : option;
```

Comment:
Why is the options.patch not in this function too? Might allow for some de duplication of the body

Reply:
I rewrote it in https://github.com/algolia/search-insights.js/pull/401/commits/87133dbd6d2ee7fc5713b9651251c5cc21440467, wdyt?